_includes/author-profile.html

```diff
@@ -34,7 +34,7 @@ <h3 class="author__name" itemprop="name">{{ author.name }}</h3>
     {% endif %}
     {% if author.bio %}
       <p class="author__bio" itemprop="description">
-        {{ author.bio }}
+        {{ author.bio | markdownify }}
```

Comment:
`author__bio` will need to be rethought to do this, perhaps changing the `<p>` element to a `<div>`. Not sure if that will break any styling though.

The reason for this is the `markdownify` filter will wrap text in `<p></p>` so you'll get something like

```html
<p class="author__bio" itemprop="description">
  <p>I am an <em>amazing</em> person.</p>
</p>
```

Which isn't great.

Reply:
I changed it to a `<div>` and successfully tested it. It looks like this:
```html
<div class="author__bio" itemprop="description">
  {{ author.bio | markdownify }}
</div>
```
which produces
```html
<div class="author__bio" itemprop="description">
  <p>I am an <strong>amazing</strong> person.</p>
</div>
```

Reply:
This doesn't break styling because the [.scss](https://github.com/mmistakes/minimal-mistakes/blob/master/_sass/minimal-mistakes/_sidebar.scss) references the `.author__bio` class:
```scss
.author__bio {
  margin: 0;

  @include breakpoint($large) {
    margin-top: 10px;
    margin-bottom: 20px;
  }
}
```